cmd/uhc/main.go

```diff
@@ -63,6 +63,7 @@ func init() {
 	root.AddCommand(token.Cmd)
 	root.AddCommand(version.Cmd)
 	root.AddCommand(cluster.Cmd)
+	root.AddCommand(completionCmd)
```

Comment:
All the other commands in the CLI live in their own package. Can you do the same for this one? Create a new `cmd/uhc/completion` directory and put the code there. Then this will be `root.AddCommand(completion.Cmd)`.

Reply:
This isn't easily done, as I need to refer back to the top-level command (`root`): https://github.com/openshift-online/uhc-cli/pull/9/files#diff-7e01ad4f5f72c98103727d222dba6a64R42

Reply:
You can do `cmd.Root()...`.

Reply:
Thanks for the tip! Fixed.